ci/npm-deploy.yml

```diff
@@ -24,7 +24,7 @@
 .packages_matrix_connect: &packages_matrix_connect
   parallel:
     matrix:
-      - PACKAGE: ["connect", "connect-web"]
+      - PACKAGE: ["connect", "connect-web", "connect-webextension"]
```

Comment:
is that really enough? 🤔 maybe yes, if yes, it would be just great :D

Reply:
According to my understanding of the code I have seen yes. 
So if yes, good job @mroz22 writing nice pipelines! :D

Reply:
i quite don't understand how is version in package.json bumped for connect-web and connect-webextension. I must be overlooking it. could you point me to the script responsible for this?

Reply:
That is a good point. It required to be added to packages/connect/scripts/bump-version.ts so it is bumped during pre-release GitHub job.

I added new commit with that https://github.com/trezor/trezor-suite/pull/11048/commits/2f3fdb44c5742e46e5b47eec3bffc220f8b7601e and tested it locally and it will work ok, but right now from `0.0.0-beta.3` it will bump it to `0.0.0` and then to `0.0.1` and so on. 

Maybe we want fist production release to be directly `1.0.0`  so we can do that change manually and then use the automation.

Reply:
> Maybe we want fist production release to be directly 1.0.0 so we can do that change manually and then use the automation.

Yes. The first release should be manual, only then we will switch to using this script

> I imagine the situation where one day we want to update connect as minor and connect-webextension as major but I think in that case we can update it manually. But the automation should work in majority of situations.

Possibly yes. this is a standalone topic however. For now we will stick to always releaseing all the 3 packages under same semver